pyproject.toml

```diff
@@ -1,3 +1,96 @@
+[tool.poetry]
+name = "epyqlib"
+version = "0.0.0"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+
+[tool.poetry-dynamic-versioning]
+#modifies epyqlib/__init__.py
+enable = true
+vcs = "git"
+style = "pep440"
+
+[tool.poetry.dependencies]
+python = "3.7.9"
+
+#### EPC Repos ####
+sunspecdemo = {git = "https://github.com/epcpower/sunspec-demo", rev = "6a07576a9d3c16f00a31121ff96edcbfe0d73822"}
+###################
+
+alqtendpy = "0.0.4"
+arrow = "0.12.1"
+appdirs = "1.4.3"
+attrs = ">=19.3.0"
+bitstruct = "6.0.0"
+boto3 = "1.14.41.0"
+boto3-type-annotations = "0.3.1"
+canmatrix = "0.9.1"
+certifi = "2020.6.20"
+certitude = { version = "1.0.1", markers = "sys_platform == 'Darwin'" }
+click = "<=7.1.2"
+dulwich = "0.20.6"
+fab = "3.0.0"
+GitPython = "2.1.15"
+graham = "0.1.11"
+marshmallow = "2.16.3"
+natsort = "5.5.0"
+paho-mqtt = "1.4.0"
+Pint = "0.11"
+pyelftools = "0.26"
+PyQt5 = ">=5.13.0"
+##pyqt5-tools fails
+##pyqt5-tools = { version = "5.14.1.3", markers = "sys_platform == 'win32'" }
```

Comment:
I think we will need to look into why `pyqt5-tools` fails. It is used to design the UI, which is necessary for development. Perhaps it should go under the `tool.poetry.dev-dependencies` section? Also, I see `PyQt5Designer` below. Does that take the place of `pyqt5-tools`? Perhaps `PyQt5Designer` should go under `tool.poetry.dev-dependencies`?

Reply:
Addressed this by downgrading `pyqt5-tools`. The only other known option would have been to use a beta version of poetry. 6783fd6ef32a903b853a4cb286b3d9f2583c0972